src/twisted/application/runner/test/mockreactor.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Mock reactor.
+"""
+
+from twisted.internet import error
+from twisted.test.proto_helpers import MemoryReactor as BaseReactor
+
+
+
+class MockReactor(BaseReactor):
```

Comment:
Could we maybe just put the `install` method on `MemoryReactor` itself, to avoid the proliferation of classes?  Or perhaps put this in `proto_helpers` itself?

If you don't feel that this is sufficiently mature to move to a central location, that's OK, but please file a refactoring ticket so that we don't lose track of this new reactor mock.